inference-engine/src/inference_engine/include/openvino/runtime/parameter.hpp

```diff
@@ -19,342 +19,14 @@
 #include <utility>
 #include <vector>
 
-#include "openvino/core/except.hpp"
-#include "openvino/runtime/common.hpp"
+#include "openvino/core/any.hpp"
 
 namespace ov {
 namespace runtime {
-
-/**
- * @brief This class represents an object to work with different parameters
- *
- */
-class OPENVINO_RUNTIME_API Parameter {
```

Comment:
This class is a part of the OV's public API, versioned and already published to the end users. Can you explain why you're just completely removing this class? What's the ticket number with a description and rationale?

Reply:
It is not removed. It is renamed as `ov::Any` and moved from 'openvino/runtime' to `openvino/core/`.
It is a part of OV 2.0  duplicating entities elimination activity. 
It should replace `ie::Prameter` and `ngraph::Variant`

Reply:
Got it, understood now.